StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+DeferredAPI.swift

```diff
@@ -17,6 +17,8 @@ extension PaymentSheet {
         authenticationContext: STPAuthenticationContext,
         paymentHandler: STPPaymentHandler,
         isFlowController: Bool,
+        isDashboardApp: Bool,
```

Comment:
Do we need this param? Can't we just check the key here directly?

Reply:
I'm not sure how I feel about adding extra params (and thus extra complexity) just for the sake of testing, plus this makes the code more tightly bound to dashboard.
Also, this param could potentially be out of sync with the key (we could use a uk and set param to false, or viceversa).